clang/test/C/C2x/n2683.c

```diff
@@ -1,4 +1,5 @@
 // NOTE: Assertions have been autogenerated by utils/update_cc_test_checks.py UTC_ARGS: --version 3
+// RUN: %clang_cc1 -triple=x86_64 -verify -ffreestanding -std=c2x %s
```

Comment:
This triple is the same as the one below (c2x and c23 are aliases), did you mean to use `-std=c11` or something along those lines?

Reply:
Nope, I just want to test c20 and c23. But yes I think I should add `std=c11/c17/...` like below

Reply:
Ah! There is no c20 release. There is C89/C90, C99, C11/C1x, C17/C18/C1y, and C23/C2x

Reply:
I see. I already remove!